core/templates/dev/head/pages/exploration_editor/statistics_tab/StatisticsTab.js

```diff
@@ -200,48 +200,22 @@ oppia.controller('StatisticsTab', [
               ];
 
               var _getVisualizationsHtml = function() {
-                htmlSnippets = visualizationsInfo.map(
-                  function(visualizationInfo) {
-                    var isAddressedResults = null;
-                    if (visualizationInfo.show_addressed_info) {
-                      var explorationId = ExplorationDataService.explorationId;
-                      var state = ExplorationStatesService.getState(
-                        $scope.stateName);
-
-                      isAddressedResults = visualizationInfo.data.map(
-                        function(datum) {
-                          var interactionId = state.interaction.id;
-                          var rulesServiceName = (
-                            AngularNameService.getNameOfInteractionRulesService(
-                              interactionId));
-                          var rulesService = $injector.get(rulesServiceName);
-                          return (
-                            AnswerClassificationService
-                              .isClassifiedExplicitlyOrGoesToNewState(
-                                explorationId, state.name, state,
-                                datum.answer, rulesService
-                              )
-                          );
-                        }
-                      );
-                    }
-
-                    var escapedData = HtmlEscaperService.objToEscapedJson(
-                      visualizationInfo.data);
-                    var escapedOptions = HtmlEscaperService.objToEscapedJson(
-                      visualizationInfo.options);
-                    var escapedIsAddressedResults =
-                      HtmlEscaperService.objToEscapedJson(isAddressedResults);
-
-                    var el = $(
-                      '<oppia-visualization-' +
-                      $filter('camelCaseToHyphens')(visualizationInfo.id) +
-                      '/>');
-                    el.attr('data', escapedData);
-                    el.attr('options', escapedOptions);
-                    el.attr('is-addressed', escapedIsAddressedResults);
-                    return el.get(0).outerHTML;
-                  });
+                var htmlSnippets = visualizationsInfo.map(function(vizInfo) {
+                  var escapedData =
+                    HtmlEscaperService.objToEscapedJson(vizInfo.data);
+                  var escapedOptions =
+                    HtmlEscaperService.objToEscapedJson(vizInfo.options);
+
+                  var el = $(
+                    '<oppia-visualization-' +
+                    $filter('camelCaseToHyphens')(vizInfo.id) + '/>');
+                  el.attr('data', escapedData);
```

Comment:
Can we call the attribute jsonData or escapedData? Ditto for options. This helps distinguish it from the additional "addressed" boolean which is not escaped.

Reply:
Done.